mempool/collector.go

```diff
@@ -300,11 +300,15 @@ func ParseTxns(txs []exptypes.MempoolTx, params *chaincfg.Params, lastBlock *Blo
 	revs := make([]exptypes.MempoolTx, 0)
 	regular := make([]exptypes.MempoolTx, 0)
 
-	var totalOut float64
+	var regularTotal, ticketTotal, voteTotal, revTotal float64
+	var likely bool
```

Comment:
Seems like the meaning is "likely to be mined".  Is that the intended semantics?

Reply:
More like "likely available to be mined" because it doesn't ever exclude tickets regardless of their count. I'm thinking `likely` -> `mineable` or `likelyMineable`

Reply:
Yeah, either way is good.